src/blocks/utils/AlchemyAPI.ts

```diff
@@ -0,0 +1,51 @@
+import { Alchemy, Network } from "alchemy-sdk";
```

Comment:
does this need to be added in the package.json? make sure that when you run just the block-sdk (not within the Seam webapp) it is still able to find the dependency -- I have a feeling that you'll need to explicitly `yarn add alchemy-sdk` to the block repo.

Reply:
added alchemy-sdk to the package.json 
ran into a problem with not having the environment variables in the standalone block-sdk so it defaulted to the demo network, which caused weird issues with the network request returning 429s. **users need to add their own .env with an alchemy api key to the block-sdk to remix the nft block**

Reply:
actually @james-buret can you add a comment that says that at the top of the file?